unified-codes/apps/web/src/selectors/ExplorerSelectors.ts

```diff
@@ -79,6 +80,39 @@ const selectRowsPerPage = createSelector(
 
 const selectPage = createSelector(selectTable, (table: IExplorerTableState): number => table?.page);
 
+const selectTypes = createSelector(
+  selectToggleBar,
+  (toggleBar: IExplorerToggleBarState): EEntityType[] =>
+    Object.keys(toggleBar).filter((type: EEntityType) => toggleBar[type]) as EEntityType[]
+);
```

Comment:
I think this is going against the design decision of having the types explicit in the store?